iothub/device/src/Transport/HttpClientHelper.cs

```diff
@@ -562,12 +562,16 @@ private static StringContent CreateContent<T>(T entity)
         private static async Task<T> ReadAsAsync<T>(HttpContent content, CancellationToken token)
         {
             token.ThrowIfCancellationRequested();
+#if NET5_0
```

Comment:
You could also model this as:

```csharp
using Stream stream = await content.ReadAsStringAsync(
#if NET5_0
    token
#endif
    );
```

I don't think you have to do that, but it is an option.